query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedConnectMutationInsideUpdateSpec.scala

```diff
@@ -806,8 +807,9 @@ class NestedConnectMutationInsideUpdateSpec extends FlatSpec with Matchers with
            |}
       """,
         project,
-        errorCode = 3042,
-        errorContains = "The change you are trying to make would violate the required relation 'ChildToParent' between Child and Parent"
+        errorCode = 0, // 3042,
+        errorContains = """Error occurred during query execution:\nInterpretationError(\"Error for binding \\'2\\': RelationViolation(RelationViolation { relation_name: \\\"ChildToParent\\\", model_a_name: \\\"Child\\\", model_b_name: \\\"Parent\\\" })"""
+          // "The change you are trying to make would violate the required relation 'ChildToParent' between Child and Parent"
```

Comment:
These error messages can probably be improved, should I spend time on that?

Reply:
Yes. That would be great :pray